ksqldb-engine/src/test/java/io/confluent/ksql/internal/StorageUtilizationMetricsReporterTest.java

```diff
@@ -103,8 +109,8 @@ public void shouldAddNodeMetricsOnConfigure() throws IOException {
     assertThat((Long) storageTotalValue, greaterThan(0L));
     assertThat((Long) storageUsedValue, greaterThan(0L));
     assertThat((Double) pctUsedValue, greaterThan(0.0));
+    assertThat((Integer) numStatefulTasksValue, greaterThan(0));
     assertThat((BigInteger) maxTaskUsageValue, greaterThanOrEqualTo(BigInteger.ZERO));
-    assertEquals(numStatefulTasksValue, 7);
```

Comment:
This test was relying on other test. When run by itself it would fail. I just adapted it to not do so